### PR TITLE
Dit 771 - Accessibility fixes

### PR DESCRIPTION
**Pull request by pllaurado**

### CODE REVIEW

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -31,6 +31,10 @@
   font-weight: bold;
 }
 
+.normal-weight {
```
Comment (ed0906) [inline app/assets/stylesheets/layout.scss]: normal-weight? surely this `bold`?

Comment (ed0906) [reply app/assets/stylesheets/layout.scss]: I believe we already have bold classes too

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_release_case.scala.html [hunk 2]
```diff
@@ -28,11 +28,11 @@
         <div class="grid-row">
             <div class="column-full">
                 <h2 class="heading-large mt-0">This case has been released to the @queue.name queue for classification</h2>
-                <hr class="mt-0"/>
 
-                <p>The case status is now open</p>
+                <p>The case status is now open.</p>
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_release_case.scala.html]: I think we need a lozenge around `open`

File: app/assets/stylesheets/layout.scss [hunk 3]
```diff
@@ -271,4 +283,16 @@ table tbody td[align="right"], table tbody th[align="right"], table thead th[ali
   &:visited {
     color: #4c2c92;
   }
+}
+
+.full-line {
+    border-bottom: 1px solid #bfc1c3;
```
Comment (ed0906) [inline app/assets/stylesheets/layout.scss]: Not sure full-line is descriptive enough
I'd maybe split this into two classes `full-width` and `border-bottom`

File: app/assets/stylesheets/layout.scss (hunk 1 above)
Comment (pllaurado) [reply app/assets/stylesheets/layout.scss]: not used ... probably I forget to remove it

**Pull request merged**